example/lib/styles/colors.dart

```diff
@@ -87,37 +75,44 @@ class AppColors {
   static const Color inputTextDimDark = Color(0xB2FCFCFD);
 
   // Utility
-  static const Color disabledBackground = Color(0x0D1A1B1F);
+  static const Color disabledBackground = Color.fromARGB(31, 26, 27, 31);
```

Comment:
@kalpeshp0310 Please use consistent formatting, as in all other places.
The  31 value can be converted to hex.